hardhat.config.ts

```diff
@@ -4,6 +4,7 @@ import "hardhat-deploy";
 import dotenv from "dotenv";
 import type { HttpNetworkUserConfig } from "hardhat/types";
 import yargs from "yargs";
+import "hardhat-gas-reporter";
```

Comment:
Nit: weird that the linter didn't say anything, I'd have liked this line more next to the other two "standalone " imports above.
```
import "@nomiclabs/hardhat-waffle";
import "hardhat-deploy";
import "hardhat-gas-reporter";

import dotenv from "dotenv";
import type { HttpNetworkUserConfig } from "hardhat/types";
import yargs from "yargs";
```